packages/turbo-workspace-convert/__tests__/managers/pnpm.test.ts

```diff
@@ -0,0 +1,246 @@
+import path from "path";
+import { setupTestFixtures } from "turbo-test-utils";
+import { Logger } from "../../src/logger";
+import MANAGERS from "../../src/managers";
+import { PackageManagers } from "../../src/types";
+import {
+  extendMatrix,
+  generateTestMatrix,
+  validateWorkspace,
+} from "../test-utils";
+import fs from "fs-extra";
+import { existsSync } from "fs-extra";
+
+const pnpm = MANAGERS.pnpm;
```

Comment:
Splitting these test files really feels like we're going to get significant drift for something that is nearly identical, or could at least be made to have identical interface assertion helpers.

Can we loop 'em in one?

Reply:
We can, pnpm tends to be it's own special snowflake for a few cases. But I'll look into consolidating